omnibus/README.md

```diff
@@ -14,7 +14,7 @@ The build pipeline generally looks like the following:
 1. When the PR is merge Expeditor takes over. It runs the list of tasks we have specified in `.expeditor/config.yml`.
 1.1. These tasks include things like automatically bumping versions, kicking off builds in [Manhattan](http://manhattan.ci.chef.co/), etc. Look at the config file for the current list of [actions](http://expeditor-docs.es.chef.io/actions/).
 1.1. The Manhattan build pipeline is configured via the [opscode-ci](https://github.com/chef-cookbooks/opscode-ci) cookbook.
-1. Notifications from Expeditor will be posted in Slack to #shake-shack (also configured in `.expeditor/config.yml`). Any failures will need to be addressed.
+1. Notifications from Expeditor will be posted in Slack to #chef-ws-notify in Chef's internal slack (also configured in `.expeditor/config.yml`). Any failures will need to be addressed.
```

Comment:
"... will be posted in _Chef's internal_ Slack to #chef-ws-notify ~in Chef's internal slack~ (also ..."